web/src/elements/cards/AggregateCard.ts

```diff
@@ -34,57 +32,59 @@ export class AggregateCard extends AKElement implements IAggregateCard {
      *
      * @attr
      */
-    @property()
-    icon?: string;
+    @property({ type: String })
+    public icon: string | null = null;
 
     /**
      * The title of the card.
      *
      * @attr
      */
-    @property()
-    header?: string;
+    @property({ type: String })
+    public label: string | null = null;
 
     /**
      * If this is non-empty, a link icon will be shown in the upper-right corner of the card.
      *
      * @attr
      */
-    @property()
-    headerLink?: string;
+    @property({ type: String })
+    public headerLink: string | null = null;
 
     /**
      * If this is non-empty, a small-text footer will be shown at the bottom of the card
      *
      * @attr
      */
-    @property()
-    subtext?: string;
-
-    /**
-     * If this is set, the contents of the card will be left-justified; otherwise they will be
-     * centered by default.
-     *
-     * @attr
-     */
-    @property({ type: Boolean, attribute: "left-justified" })
-    leftJustified = false;
+    @property({ type: String })
+    public subtext: string | null = null;
 
     static styles: CSSResult[] = [
         PFBase,
         PFCard,
         PFFlex,
         css`
+            .pf-c-card {
+                container-type: inline-size;
+            }
+
             .pf-c-card.pf-c-card-aggregate {
                 height: 100%;
             }
             .pf-c-card__header {
-                flex-wrap: nowrap;
+                padding: var(--pf-global--spacer--md);
             }
-            .center-value {
-                font-size: var(--pf-global--icon--FontSize--lg);
-                text-align: center;
+            .pf-c-card__title {
+                display: flex;
+                align-items: center;
+                gap: var(--pf-global--spacer--sm);
+                flex: 1 1 auto;
+
+                @container (width < 200px) {
+                    font-size: var(--pf-global--FontSize--sm);
+                }
             }
+
```

Comment:
Fixes some odd sizing and positioning discovered while adjusting the card elements to be ARIA-friendly.